src/librustdoc/html/static/main.js

```diff
@@ -2740,6 +2740,10 @@ function defocusSearchBar() {
     }
 
     window.addSearchOptions = function(crates) {
+        if (search_input) {
```

Comment:
Even if very unlikely, I don't like the idea to enable the search input if the other processing isn't done. So instead of just moving this call here, I suggest to put it inside a function and then call this function in the `if (!elem) {` block and at the end of the `addSearchOptions`. What do you think?

Reply:
Thanks, yes I didn't think about the processing angle, made the change now.

Reply:
👍